xarray/tests/test_dataarray.py

```diff
@@ -1921,9 +1921,9 @@ def test_inplace_math_basics(self):
     def test_inplace_math_automatic_alignment(self):
         a = DataArray(range(5), [("x", range(5))])
         b = DataArray(range(1, 6), [("x", range(1, 6))])
-        with pytest.raises(xr.MergeError):
+        with pytest.raises(ValueError):
```

Comment:
I would suggest adding `match` here to match some text in the new exception, to verify that it's working, i.e.,
```suggestion
        with pytest.raises(ValueError, match="Automatic alignment is not supported"):
```